packages/clarity-js/src/insight/blank.ts

```diff
@@ -9,3 +9,4 @@ export function stop(): void {}
 export function log(): void {}
 export function observe(): void {}
 export function compute(): void {}
+export function hashText(): void {}
```

Comment:
you might want to create a performance/blank and add this there instead so it doesn't affect the insights package